src/Accordion/AccordionPanel.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import StyledAccordionPanel from './StyledAccordionPanel';
+
+const AccordionPanel = ({ isOpen, ...panelProps }) => {
+  const display = isOpen ? 'block' : 'none';
```

Comment:
Can/should this be done with toggling css classes instead? Old logic I believe would say toggling css classes is faster but maybe that's no longer the case.

Reply:
@mikecx It might worth noting that this isn’t setting an inline style, but rather results in a CSS class name because of emotion. We’re utilizing the `styled-system`’s `display` prop for it.

Reply:
Aha, cool. I'll need to spend a bit more time in Arbor soon to get back up to speed.